### ISSUE TITLE
`Rfree` should be `Rfree`, not `Rport`

### ISSUE THREAD

**Issue by Mv77** [econ-ark/HARK#1158]
The current implementation of models with risky assets includes the following behavior in `get_Rfree`:
https://github.com/econ-ark/HARK/blob/a462169fef6c047a070239da749a4f449ecf6c89/HARK/ConsumptionSaving/ConsRiskyAssetModel.py#L270-L275

I get that the purpose of this is to recycle the transition code of models without a risky asset, sneaking in an `Rfree` that is not actually the risk-free rate but the return implied by the optimal portfolio instead. However I do not think it is a good thing that a future user that is implementing a model and wants to get the risk-free rate and naively calls `get_Rfree` gets something back that is **not** `Rfree` and is forced to go hunting for the `super()` that causes that; that happened to me just now.

Maybe the right thing to do is to have an `Rport` that is set to `Rfree` in models without risky assets. Not the other way around.

**Comment by sbenthall** [comment]
Totally agree that this is a problem.

**Comment by llorracc** [comment]
Yes, Rport degenerating to Rfree much better than Rfree being stochastic.

**Comment by llorracc** [comment]
Reviewing this again: I would have thought that our defaults would be for a zero percent portfolio allocation to the risky asset, and maybe zero variance for the risky shocks.  So am surprised that @Mv77 found that when he didn't touch anything the `Rport` he got was actually stochastic.

**Comment by mnwhite** [comment]
Strongly agree. Or we need a better variable name for "realized rate of
return on assets" in the simulation code other than Rport, and the method
is called get_Rbettername()

**Comment by mnwhite** [comment]
The expanded Monte Carlo framework in #1545 simplifies the simulation code so that misleadingly name methods like this can (eventually) go away.
